synapse/lib/lmdb.py

```diff
@@ -1,13 +1,11 @@
 import sys
 import struct
-import itertools
+
+import xxhash  # type: ignore
```

Comment:
Does this type annotation need to be added to third party code or just modules w/o mypy stubs?

Reply:
Just the modules without mypy stubs.